how-to/secure/enable-tls-clients.md

```diff
@@ -156,3 +184,6 @@ The TLS versions, key exchange protocols and encryption algorithms supported by
 - TLS\_ECDHE\_ECDSA\_WITH\_AES\_256\_GCM\_SHA384
 - TLS\_ECDHE\_RSA\_WITH\_CHACHA20\_POLY1305
 - TLS\_ECDHE\_ECDSA\_WITH\_CHACHA20\_POLY1305
```

Comment:
hi, could you help remove those week cipher suits https://github.com/pingcap/docs-cn/pull/2998/files in PR by the way~? thx

Reply:
Updated~

Reply:
This PR is cherry-picked to v3.1 and v4.0, yet https://github.com/pingcap/docs-cn/pull/2998 is cherry-picked to v3.0, v3.1 and v4.0. So I open https://github.com/pingcap/docs/pull/2504 to remove those week cipher suits from v3.0~